app-editors/featherpad/featherpad-1.2.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Can we use `EAPI=8` here?

Reply:
Sure, I'll push an update on Monday. Or, if proxy-maint will fix it, I don't mind.

Reply:
I have force pushed the EAPI update.